reference/conanfile/other.rst

```diff
@@ -20,13 +20,30 @@ Check the source code. You might be able to produce different outputs with diffe
 Running commands
 ----------------
 
+
+.. code-block:: python
+
+    run(self, command, output=True, cwd=None, win_bash=False, subsystem=None, msys_mingw=True):
+
+
 ``self.run()`` is a helper to run system commands and throw exceptions when errors occur,
 so that command errors are do not pass unnoticed. It is just a wrapper for ``os.system()``
 
 Optional parameters:
 
-- ``output``: Defaulted to True, will write in stdout. You can pass any stream that accepts a ``write`` method like a ``StringIO.StringIO()``
-  Note: Be careful with the Python2 and Python3 compatibility, StringIO has been realocated between them.
+- **output**:(Optional, Defaulted to ``True``) When True it will write in stdout.
```

Comment:
Fix this (and following) to -->    **output** (Optional, Defaulted to ``True``):